util/mdbook_otbn.py

```diff
@@ -68,7 +68,7 @@ def main() -> None:
     def ref_to_link(m: re.Match):
         instr = m.group(1)
         ref = instr.replace(".", "").lower()
-        return '<a href="/{}#{}"><code>{}</code></a>)'.format(isa_book_path, ref, instr)
+        return '<a href="/book/{}#{}"><code>{}</code></a>'.format(isa_book_path, ref, instr)
```

Comment:
Thanks for this! If it feels like getting the other commits of this PR will take a few days to merge, it might make sense to open up a parallel PR with just this commit (since there's definitely no question about whether to merge this!)